lib/fathead/spring_framework/fetch.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+# see http://stackoverflow.com/a/10959815/834
+rm -rf download
+rm -rf docs
+mkdir -p download
+mkdir -p docs
+cd ./download/
+unamestr=`uname`
+if [[ "$unamestr" == 'Darwin' ]]; then
```

Comment:
I have commented out the os selection and curl.  Let me know if you want to remove it completely.